{{cookiecutter.repostory_name}}/app/Dockerfile

```diff
@@ -1,19 +1,41 @@
-FROM python:3.6
+{% if cookiecutter.use_alpine_linux == "y" -%}
+FROM python:3.6-alpine AS base-image
+RUN apk add postgresql-dev gcc python3-dev musl-dev
+{% else -%}
+FROM python:3.6 AS base-image
+{% endif -%}
 
 COPY ./bin/s6-overlay-amd64.tar.gz /tmp/s6-overlay-amd64.tar.gz
 
 RUN tar xvfz /tmp/s6-overlay-amd64.tar.gz -C / \
     && rm -f /tmp/s6-overlay-amd64.tar.gz {% if cookiecutter.use_celery == "y" %}\
     && adduser --disabled-password --gecos '' celery{% endif %}
 
+RUN pip3 install --user --no-cache-dir psycopg2-binary==2.8.3 gunicorn==19.9.0
+
+ENV PATH=/root/.local/bin:$PATH
+
+COPY ./src/requirements.txt /srv/app/requirements.txt
+
+WORKDIR /srv/app
+
+RUN pip3 install --user --no-cache-dir -r /srv/app/requirements.txt
+
+
+{% if cookiecutter.use_alpine_linux == "y" -%}
+FROM python:3.6-alpine AS secondary-image
+{% else -%}
+FROM python:3.6 AS secondary-image
+{%- endif %}
+COPY --from=base-image /root/.local /root/.local
+
 COPY ./src/ /srv/app/
 
 WORKDIR /srv/app
 
-RUN pip3 install --no-cache-dir -r requirements.txt gunicorn\
-    && python3 -m compileall -b .
+ENV PATH=/root/.local/bin:$PATH
 
-ENTRYPOINT ["/init"]
```

Comment:
I can't understand what is or was `/init`. I can't find it anywhere in this repo or `python` image itself.

Reply:
that looks like a part of s6-overlay